config/samples/external_secret.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
   labels:
     acme.org/owned-by: "q-team"
   annotations:
-    acme.org/sha: 1234
+    acme.org/sha: "1234"
```

Comment:
for some reason without this it caused a change in the csv. (i suspect minor differences in tool versions)